outport/process/transactionsfee/dataHolders_test.go

```diff
@@ -39,13 +40,13 @@ func TestTransactionsAndScrsHolder(t *testing.T) {
 				},
 			},
 
-			scrHash2: {
+			hex.EncodeToString([]byte(scrHash2)): {
 				SmartContractResult: &smartContractResult.SmartContractResult{},
 				FeeInfo: &outportcore.FeeInfo{
 					Fee: big.NewInt(0),
 				},
 			},
-			scrHash3: {
+			hex.EncodeToString([]byte(scrHash3)): {
```

Comment:
We could've made txHash, scrHash1,2,3 to be []byte, so we don't have to do the encoding each time in these tests